pkg/engine/anchor/common.go

```diff
@@ -1,105 +1,118 @@
 package anchor
 
 import (
+	"errors"
 	"path"
+	"regexp"
 	"strings"
 )
 
 // IsAnchor is a function handler
 type IsAnchor func(str string) bool
+type AnchorType string
+
+const (
```

Comment:
does this need to be public ?

if not, we could just use a string.

Reply:
I initially made this public because i was thinking of using it as a type as an argument for hasAnchor function